test/install.c

```diff
@@ -1496,7 +1497,7 @@ int main(int argc, char *argv[])
 		install_data = memdup((&(InstallData) {
 			.message_needles = memdup((&(const gchar *[]) {
 				"Checking and mounting bundle...",
-				"Debug: --dummy1 --dummy2",
+				"Debug: arg1 arg2 --dummy1 --dummy2",
```

Comment:
How do the multiple sources of arguments interact? That should be documented.

Reply:
I have not found a good way to provide more verbose argument descriptions. Maybe the man page or a separate reference chapter would be a proper place. For now I have added a short hint in the manifest reference about arguments.
I have now also clarified the purpose of ``--handler-args``  in the help output a bit.